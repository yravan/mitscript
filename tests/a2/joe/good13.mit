
f = fun() {
  print("Inside f, which doesn't have a return statement");
};

res = f();
print("Should print None: " + res);
