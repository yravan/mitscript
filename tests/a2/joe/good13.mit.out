Inside f, which doesn't have a return statement
Should print None: None
